SDK_versioned_docs/version-3.0.0/widgets/swap-widget-api.md

```diff
@@ -0,0 +1,50 @@
+---
```

Comment:
If you can, I think you should omit the right-hand ToC sidebar from this view. It doesn't add much, but it does reduce the view enough that I have horizontal overflow and cannot easily see all of the docs.

Reply:
it's a very long doc, i think ToC is helpful